addons/stock_account/tests/test_stockvaluation.py

```diff
@@ -103,6 +103,56 @@ def _get_stock_valuation_move_lines(self):
             ('account_id', '=', self.stock_valuation_account.id),
         ], order='date, id')
 
+
+    def _make_in_move(self, product, quantity, unit_cost=None):
+        """ Helper to create and validate a receipt move.
+        """
+        unit_cost = unit_cost or product.standard_price
+        in_move = self.env['stock.move'].create({
+            'name': 'in %s units @ %s per unit' % (str(quantity), str(unit_cost)),
+            'product_id': product.id,
+            'location_id': self.env.ref('stock.stock_location_suppliers').id,
+            'location_dest_id': self.env.ref('stock.stock_location_stock').id,
+            'product_uom': self.env.ref('uom.product_uom_unit').id,
+            'product_uom_qty': quantity,
+            'price_unit': unit_cost,
+            'picking_type_id': self.env.ref('stock.picking_type_in').id,
+        })
+
+        in_move._action_confirm()
+        in_move._action_assign()
+        in_move.move_line_ids.qty_done = quantity
+        in_move._action_done()
+
+        return in_move.with_context(svl=True)
+
+    def _make_out_move(self, product, quantity):
+        """ Helper to create and validate a delivery move.
+        """
+        out_move = self.env['stock.move'].create({
+            'name': 'out %s units' % str(quantity),
+            'product_id': product.id,
+            'location_id': self.env.ref('stock.stock_location_stock').id,
+            'location_dest_id': self.env.ref('stock.stock_location_customers').id,
+            'product_uom': self.env.ref('uom.product_uom_unit').id,
+            'product_uom_qty': quantity,
+            'picking_type_id': self.env.ref('stock.picking_type_out').id,
+        })
+        out_move._action_confirm()
+        out_move._action_assign()
+        out_move.move_line_ids.qty_done = quantity
+        out_move._action_done()
+        return out_move.with_context(svl=True)
+
+    def _get_new_fifo_product(self):
+        product = self.env['product.product'].create({
+            'name': 'product1',
+            'type': 'product',
+            'categ_id': self.env.ref('product.product_category_all').id,
+        })
+        product.product_tmpl_id.categ_id.property_cost_method = 'fifo'
+        return product
```

Comment:
a shame to introduce these helpers in this file and they are used in 3 of the 50 tests :( can you remove them et create the move the same way the others do? so this file is consistent